utils/manifest-generator/generate-llbuild-manifest

```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/env python3
+
+import sys
+import argparse
+import uuid
+import random
+
+class Command:
+	def __init__(self, name, index, inputList):
+		self.name = name
+		self.outputFile = name
+		self.index = index
+		self.inputList = inputList
+		self.args = ""
+		self.hasDependantCommand = False
+
+	def setupArgs(self):
+		if not self.inputList:
+			self.args = "echo " + self.name + " > " + self.outputFile
+		else:
+			self.args = "echo " + self.name + " > " + self.outputFile + " && " + "cat " + " ".join(input for input in self.inputList) + " >> " + self.outputFile
+
+	def addNewInput(self, newInput):
+		self.inputList.append(newInput)
+
+class LLBuildManifest:
+	def __init__(self, path, clientName, clientVersion, density, maxAdditionalInputs=sys.maxsize, minDepth = -1, maxDepth = -1, minWidth = -1, maxWidth = -1, levelWidths = []):
+		print("Writing llbuild manifest to " + path)
+		print("Initial Setup")
+
+		self.buffer = []
+		try:
+			self.file = open(path, 'w+')
+			self.nextCommandID = 0
+			
+			self.levelWidths = levelWidths
+			self.setupLevelWidths(minDepth, maxDepth, minWidth, maxWidth)
+
+			self.commands = []
+			self.clientName = clientName
+			self.clientVersion = clientVersion
+			self.density = density
+			self.maxAdditionalInputs = maxAdditionalInputs
+		except:
+			raise Exception
```

Comment:
I would recommend not catching this execution like this, it just hides the actual error.